samples/csharp_dotnetcore/45.state-management/Startup.cs

```diff
@@ -28,15 +27,36 @@ public void ConfigureServices(IServiceCollection services)
             // Create the Bot Adapter with error handling enabled.
             services.AddSingleton<IBotFrameworkHttpAdapter, AdapterWithErrorHandler>();
 
+            /* JSON SERIALIZER - Uncomment the code in this section to use a JsonSerializer with a custom SerializationBinder configuration. */
+            // Note: the AllowedTypesSerializationBinder limits the objects the storage is able to read and write, by providing a list of types used to allow or deny. It can be used to increase security.
+
+            // var jsonSerializer = JsonSerializer.Create(new JsonSerializerSettings
+            // {
+            //     TypeNameHandling = TypeNameHandling.All,
+            //     MaxDepth = null,
+            //     SerializationBinder = new AllowedTypesSerializationBinder(
+            //         new List<Type>
+            //         {
+            //             typeof(Dictionary<string, object>),
+            //             typeof(ConversationData),
+            //             typeof(UserProfile)
+            //         })
+            // });
+
+            /* JSON SERIALIZER - Uncomment the code in this section to use a JsonSerializer with a custom SerializationBinder. */
```

Comment:
I would just add "configuration" so that both texts match.

Reply:
yea, nice catch, i forgot to update the other one

Reply:
done, thanks!